src/functions-and-arrays.js

```diff
@@ -52,14 +91,31 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
-
+//function uniquifyArray()
+function uniquifyArray(words) {
+  let uniqueWords = uniquifyArray(wordsUnique) ;
+  for (let i = 0; i < words.length; i++) {
+      if (uniqueWords.indexOf(words[i]) === -1) {
+          uniqueWords.push(words[i]);
+      }
+  }
+  return uniqueWords;
+}
+console.log (uniquifyArray);
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
+function doesWordExist(wordArray, searchWord) {
+  if(wordArray.includes(searchWord)) {
```

Comment:
You also missed checking if it receives an empty array when called.

`if(!wordArray.length) return null;`